ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -118,6 +118,11 @@ function AuthProviders(): ReactElement {
         history.push(getEntityPath(entityType, entityId, { ...queryObject, action: undefined }));
     }
 
+    function getProviderLabel(): string {
+        const provider = availableProviderTypes.find(({ value }) => value === type) ?? {};
+        return (provider.label as string) ?? 'auth';
```

Comment:
Out of curiosity, what does TypeScript think about `provider?.label ?? 'auth'`

Reply:
It was not a fan. `provider` comes from a value stored in Redux and the selector is still .js, so it has an `any` type.

That change gives an `unsafe return of 'any' type` error. This seemed like a less invasive way to get this value to typecheck, versus defining types for the source of the data.

Reply:
Good call.

Out of scope for this PR, there are 4 occurrences of type parameter for `createStructuredSelector` call.